throttling/pom.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The license is incorrectly positioned

Reply:
Can you elaborate. I guess the first line has to be this for pom to work correctly.